server/routes/testutils/testData.ts

```diff
@@ -199,15 +202,20 @@ export default class TestData {
 
   static notificationCount = ({ count = 5 }: Partial<NotificationCount> = {}): NotificationCount => ({ count })
 
+  // data with notification types processed
   static notificationGroup = ({
     reference = 'ab*cd*ef*gh',
-    type = 'NON_ASSOCIATION_EVENT',
-    affectedVisits = [
-      this.notificationVisitInfo(),
-      this.notificationVisitInfo({ bookedByName: 'User Two', bookedByUserName: 'user2', prisonerNumber: 'A5678DE' }),
-    ],
+    type = 'PRISONER_RELEASED_EVENT',
+    affectedVisits = [this.notificationVisitInfo()],
   }: Partial<NotificationGroup> = {}): NotificationGroup => ({ reference, type, affectedVisits })
 
+  // raw data with types as returned from API
+  static notificationGroupRaw = ({
+    reference = this.notificationGroup().reference,
+    type = this.notificationGroup().type as NotificationTypeRaw,
+    affectedVisits = this.notificationGroup().affectedVisits,
+  }: Partial<NotificationGroupRaw> = {}): NotificationGroupRaw => ({ reference, type, affectedVisits })
+
```

Comment:
Here (and with `visitBookingDetails` below), I'm making two similar test data methods to distinguish between the raw data - for use in API client and Wiremock - and processed data for use everywhere else.